cvpysdk/organization.py

```diff
@@ -963,6 +961,38 @@ def tenant_operator(self):
         tenant_operators = self._organization_info.get('organizationProperties', {}).get('operators', [])
         return [role['user']['userName'] for role in tenant_operators]
 
+    def add_user_groups_as_operator(self, user_group_list, request_type):
+        """Update the local user_group as tenant operator of the company
+
+        Args:
+            user_group_list		(Str)  -- user group list

```

Comment:
can you please change accepted type as list

Reply:
Done!